### PR TITLE
update classic bootnodes

### PR DESCRIPTION
**Pull request by stevanlohja**
## PR description

Added ETC Cooperative boot nodes which can be seen [here](https://etcnetstats.com/).

## Fixed Issue(s)

Checked for valid boot nodes.

## Changelog

cosmetic change

### CODE REVIEW

**Comment by shemnon** [comment]
run `./gradlew spotlessApply` and it will magically be formatted.

File: ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/config/DiscoveryConfiguration.java [hunk 1]
```diff
@@ -101,7 +101,33 @@
   public static final List<EnodeURL> CLASSIC_BOOTSTRAP_NODES =
       Collections.unmodifiableList(
           Stream.of(
+                  // ETC Cooperative
+                  "enode://8e73168affd8d445edda09c561d607081ca5d7963317caae2702f701eb6546b06948b7f8687a795de576f6a5f33c44828e25a90aa63de18db380a11e660dd06f@159.203.37.80:30303",
```
Comment (shemnon) [inline ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/config/DiscoveryConfiguration.java:105]: nit: could each sub-group be sorted by client-id?  It makes duplicate checking way easier.  Most IDEs have a sort selected lines option.

**Review by shemnon — APPROVED** [review]
Let me know when it's ready to merge.

**Comment by stevanlohja** [comment]
> run `./gradlew spotlessApply` and it will magically be formatted.

Completed this change.

File: ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/config/DiscoveryConfiguration.java (hunk 1 above)
Comment (stevanlohja) [reply ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/config/DiscoveryConfiguration.java:105]: Not sure what you mean. I just use Sort Lines from VS code to find and/ or remove any duplicates and manually checked them.

**Comment by stevanlohja** [comment]
Ready to merge.

**Pull request merged**